test/testinput/bufr_ncep_sfcshp.yaml

```diff
@@ -0,0 +1,222 @@
+
+          heightOfBaseOfCloud:
+            query: "*/CLDSQ1/HOCB"
+
+          cloudCoverTotal:
+            query: "*/TOCC"
+
+          heightAboveSurfaceofBaseofLowestCloud:
+            query: "*/HBLCS"
+    
+    ioda:
+      backend: netcdf
+      obsdataout: "./testrun/gdas.t12z.sfcshp.tm00.bufr_d.2020110112.nc"
+
+#      dimensions:
+#        - name: nchans
+#          paths:
+#            - "*/BRIT"
+#            - "*/BRITCSTC"
+
+#      globals: 
+#        - name: "MetaData/dataType"
+#          type: string
+#          value: "sfcshp"
+            
+      variables:
+        - name: "MetaData/dateTime"
+          source: variables/timestamp
+          longName: "dateTime"
+          units: "seconds since 1970-01-01T00:00:00Z"
+
+        - name: "MetaData/latitude"
+          source: variables/latitude
+          longName: "Latitude"
+          units: "degree_north"
+          range: [-90, 90]
+
+        - name: "MetaData/longitude"
+          source: variables/longitude
+          longName: "Longitude"
+          units: "degree_east"
+          range: [-180, 180]
+
+        - name: "MetaData/reportIdentifier"
+          source: variables/reportIdentifier
+          longName: "Report Identifier"
+          units: ""
+
+        - name: "MetaData/stationElevation"
+          coordinates: "longitude latitude"
+          source: variables/stationElevation
+          longName: "Height of Station"
+          units: "m"
+
+        - name: "ObsValue/stationPressure"
+          coordinates: "longitude latitude"
+          source: variables/stationPressure
+          longName: "Station Pressure"
+          units: "Pa"
+          range: [90000, 105000]
+
+        - name: "ObsValue/presentWeather"
+          coordinates: "longitude latitude"
+          source: variables/presentWeather
+          longName: "Present Weather"
+          units: ""
+
+        - name: "ObsValue/airTemperature"
+          source: variables/airTemperature
+          longName: "Temperature of Air"
+          units: "K"
+          range: [230, 315]
+
+        - name: "ObsValue/dewpointTemperature"
+          coordinates: "longitude latitude"
+          source: variables/dewpointTemperature
+          longName: "Dewpoint Temperature"
+          units: "K"
+          range: [200, 315]
+
+        - name: "MetaData/waterTemperatureMethod"
+          coordinates: "longitude latitude"
+          source: variables/waterTemperatureMethod
+          longName: "Method of Sea Surface Measurement"
+          units: ""
+
+        - name: "ObsValue/seaSurfaceTemperature"
+          coordinates: "longitude latitude"
+          source: variables/sst
+          longName: "Sea Surface Temperature"
+          units: "K"
+          range: [260, 315]
+
+        - name: "ObsValue/seaSurfaceSalinity"
+          coordinates: "longitude latitude"
+          source: variables/sss
+          longName: "Sea Surface Salinity"
+          units: "parts/1000"
+          range: [20, 50]
+
```

Comment:
I did not find seaSurfaceTemperature in convention table, please add it in ObsSpace.yaml. 
units: "parts/1000" => g/kg. If the units is not g/kg, you will need to use transforms to convert.